tests/test_route.py

```diff
@@ -839,6 +846,60 @@ def test_RouteAndNexthopInDifferentVrf(self, dvs, testlog):
         dvs.servers[3].runcmd("ip route del default dev eth0")
         dvs.servers[3].runcmd("ip address del 20.0.1.2/24 dev eth0")
 
+    def test_RouteAddRemoveIpv4BlackholeRoute(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        self.clear_srv_config(dvs)
+
+        # add route entry
+        dvs.runcmd("vtysh -c \"configure terminal\" -c \"ip route 2.2.2.0/24 blackhole\"")
```

Comment:
I think there are two reasons not to do it in this way: 1. It seems that bgpcfgd is not running in docker-sonic-vs. Without bgpcfgd the static route table in config_db will not be applied. 2. I feel these vs tests mainly aim to test the orchagent functionalities. If we choose to add it in config_db, it would include the bgpcfgd functions which are not currently in the scope of dvs tests.

Reply:
Agree to point 1. But for 2, I would say then we could directly add to APP_DB, why even `vtysh`. IMO, if the infra is available, lets use that path to configure.

Reply:
Okay. Not sure if we are planning to update the infra to include the support configuring from config_db in the near future. If not, maybe we should keep it as is and change the configuration after the infra is ready.